website/assets/css/homepage.css

```diff
@@ -49,6 +82,85 @@ h1, h2, h3, h4, h5, h6 {
   border-color: #fff;
 }
 
+.btn {
+    margin: 0 auto;
+    background-color: #1EC3FD;
+    color: #000;
+    text-align: center;
+    width: ;
+    font-weight: 500;
+    font-family: AvenirNext forINTUIT;
+    white-space: nowrap;
+    vertical-align: middle;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    border: 2px solid #1EC3FD;
+    padding: .4rem 5rem .4rem 5rem;
+    font-size: 1rem;
+    line-height: 1.2;
+    border-radius: 5rem;
+    transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+    margin-bottom: 30px
+}
+
+.btn:hover {
+  color: #fff;
+  background-color:transparent;
+  border-color:#;
+}
+
+.btn-outline-inverse {
+  color: #135CC0;
+  background-color: transparent;
+  border-color: #135CC0;
+  padding-top: 10px; 
+  padding-bottom: 15px;
+}
+
+.btn-outline-inverse:hover {
+  color: #ccc;
+  background-color: transparent;
+  border-color: #ccc;
+}
+
+.justify-content-center {
+
+  justify-content:center!important;
+}
+
+.container-fluid {
+    width: 100%;
+    padding-right: 0px;
+    padding-left: 0px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.feature-box {
+  padding:15px 20px 15px 70px;
+}
+
+@media (min-width:768px) {
+  .col-md-5 {
+    -ms-flex:0 0 41.66667%;
+    flex:0 0 41.66667%;
+    max-width:41.66667%;
+  }
+}
+
+.feature-box .icon {
+  font-size:30px;
+  position:absolute;
+  left:15px;
+  top:15px;
+  width:30px;
+  text-align:center;
+  color:#3b99e0;
+}
+
```

Comment:
may want to clean-up extra whitespace here

Reply:
No, It doesn't need.